docs/releasenotes/index.rst

```diff
@@ -4,7 +4,7 @@ Release Notes
 
 .. note:: If you already have a tutorial environment from a previous release see :ref:`upgrading-vagrant-environment`.
 
-v0.9
+v1.0
```

Comment:
There is no process. However, I can say that the notes here have been accepted (though maybe there should be more of them).